staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1910,9 +1910,17 @@ type PermittedHostDevices struct {
 // PciHostDevice represents a host PCI device allowed for passthrough
 // +k8s:openapi-gen=true
 type PciHostDevice struct {
-	PCIVendorSelector        string `json:"pciVendorSelector"`
-	ResourceName             string `json:"resourceName"`
-	ExternalResourceProvider bool   `json:"externalResourceProvider,omitempty"`
+	PCIVendorSelector string `json:"pciVendorSelector"`
+	// The vendor_id:product_id tupple of the PCI device
```

Comment:
I think you have put by incident the comment for the fields below the fields than above them.

Reply:
oh dear. thanks for the babysitting.